code/modules/mob/interactive.dm

```diff
@@ -865,7 +867,7 @@
 	if(prob(10)) // 10% chance to broadcast it over the radio
 		chatmsg = ";"
 
-	if(prob(chattyness) || knownStrings.len < 10) // say a generic phrase, otherwise draw from our strings.
+	if(prob(chattyness) && knownStrings.len < 10) // say a generic phrase, otherwise draw from our strings.
```

Comment:
This is supposed to be ||, not &&. 

It's making sure that if there's less than 10 known strings, we throw in some randomised ones so we don't always get the same ten lines, as well as being a fallback.